application/comit_node/src/http_api/rfc003/handlers/get_swaps.rs

```diff
@@ -37,17 +38,27 @@ pub fn handle_get_swaps<T: MetadataStore<SwapId>, S: StateStore>(
                         let communication = state.swap_communication.clone().into();
                         let alpha_ledger = state.alpha_ledger_state.clone().into();
                         let beta_ledger = state.beta_ledger_state.clone().into();
+                        let parameters = SwapParameters::new(state.clone().request());
+                        let actions: Vec<String> =
+                            state.actions().iter().map(|action| action.name()).collect();
                         let error = state.error;
                         let status =
                             SwapStatus::new(&communication, &alpha_ledger, &beta_ledger, &error);
 
                         let swap = EmbeddedSwapResource {
                             status,
                             protocol: RFC003.into(),
+                            parameters,
                         };
 
                         let mut hal_resource = HalResource::new(swap);
                         hal_resource.with_link("self", swap_path(id));
+
+                        for action in actions {
+                            let route = format!("{}/{}", swap_path(id), action);
```

Comment:
:question: Do we not have a method somewhere that generates these? We shouldn't duplicate that ...